src/js/structs/LogBuffer.js

```diff
@@ -24,10 +22,7 @@ class LogBuffer extends List {
   constructor(options = {}) {
     super(...arguments);
 
-    this.configuration = _.defaults(
-      _.pick(options, ...Object.keys(DEFAULT_OPTIONS)),
-      DEFAULT_OPTIONS
-    );
+    this.configuration = Object.assign({}, DEFAULT_OPTIONS, options);
```

Comment:
This is not doing exactly the same thing. The `pick` is limiting the the applied properties from `options`